### ISSUE TITLE
Image thumbnails

### ISSUE THREAD

**Issue by artyomn** [codekidX/storage-chooser#36]
I think it would be a good improvement to add an option to show image files' thumbnails instead of type icon.

**Comment by codekidX** [comment]
Hi, it's a nice idea but don't you think it will be heavy on the library side? 

Because the main motive is to choose directory/file as quick as possible and if we even use some lazy-loading image libraries like Glide it'll still cause some lag on low-end devices _(I think)_. Do let me know what you think about this.

**Comment by artyomn** [comment]
I think some lag may be, but it shouldn't be essential. As of my experience with Glide it is very well optimized (I've done a GridView with about 300 images loaded with Glide - and it scrolls absolutely smoothly (even if you scroll fast) on 3 years old Nexus 5).
Probably it may be done configurable - some option to show thumbnails or just icons.
I've thought about this idea (image thumbnails) because image file names usually have no sense for user but previews have.

**Comment by codekidX** [comment]
True, I'll try to include Glide in the project and implement the thumbnails asap. Thanks for your view on this feature.
